### PR TITLE
Overlay Popup Behavior Correction

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
The jumping is because of transitive. I couldn't think of a good way to stop transitive from jumping and then thought about what the desired behavior would be without the jumping (just randomly flashing colors) and it also feels really weird so I think the solution is to just disable hover effects altogether. 

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/map/itinerary-summary-overlay.tsx [hunk 1]
```diff
@@ -152,14 +152,18 @@ const ItinerarySummaryOverlay = ({
...
 
   if (!itins || !visible) return <></>
-  const mergedItins: ItinWithGeometry[] = addTrueIndex(
-    doMergeItineraries(itins).mergedItineraries.map(addItinLineString)
+  const indexedItins: ItinWithGeometry[] = addTrueIndex(
+    itins.map(addItinLineString)
+  )
+  const mergedItins = doMergeItineraries(indexedItins).mergedItineraries
+
+  const midPoints = mergedItins.reduce(
```
Comment (daniel-heppner-ibigroup) [inline lib/components/map/itinerary-summary-overlay.tsx]: this has an any type. I think `reduce` needs to have a type and `mergedItins` needs to be typed to not get errors.
Either type `narrative-itineraries.js` and `doMergeItineraries()` or assign a type to the variable here.

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]
typescript :(

File: lib/components/map/itinerary-summary-overlay.tsx (hunk 1 above)
Comment (daniel-heppner-ibigroup) [reply lib/components/map/itinerary-summary-overlay.tsx]: it still has an any type
image

**Review by daniel-heppner-ibigroup — APPROVED** [review]
fine now because I fixed it

**Review by josh-willis-arcadis — APPROVED** [review]
Changes look good. Looks like percy tests are failing visual tests? Not sure what is happening there.

**Comment by miles-grant-ibigroup** [comment]
> Changes look good. Looks like percy tests are failing visual tests? Not sure what is happening there.

Daylight savings!

**Pull request merged**